pyproject.toml

```diff
@@ -39,7 +39,7 @@ select = ["E", "W", "F", "C", "N", "R", "D", "H"]
 # Ignore D107 Missing docstring in __init__
 ignore = ["W503", "E501", "D107"]
 # D100, D101, D102, D103: Ignore missing docstrings in tests
```

Comment:
Can you please add description of the ignored tests as a comment, I see D104, D205, D212 and D415 are missing such.

Reply:
Sure.